### PR TITLE
[UseRubyVersionV0] Migrated to NodeJS10

### PR DESCRIPTION
**Pull request by sdobrodeev**
**Task name**: UseRubyVersionV0

**Description**: Migrated UseRubyVersionV0 to Node 10 execution handler.

**Documentation changes required:** No

**Added unit tests:** No

**Attached related issue:** No

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/UseRubyVersionV0/package.json [hunk 1]
```diff
@@ -23,10 +23,14 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "@types/node": "^6.0.101",
+    "@types/node": "^10.17.0",
     "@types/q": "^1.0.7",
-    "azure-pipelines-task-lib": "^2.9.3",
+    "@types/mocha": "^5.2.7",
+    "azure-pipelines-task-lib": "^3.0.6-preview.0",
     "azure-pipelines-tasks-utility-common": "^2.0.0",
```
Comment (egor-bryzgalov) [inline Tasks/UseRubyVersionV0/package.json:30]: Let's update it to `3.0.0-preview.0`

File: Tasks/UseRubyVersionV0/package.json [hunk 2]
```diff
@@ -23,10 +23,14 @@
...
     "azure-pipelines-tool-lib": "^0.12.0"
```
Comment (egor-bryzgalov) [inline Tasks/UseRubyVersionV0/package.json:31]: Let's switch to `1.0.0-preview.0`

**Comment by anatolybolshakov** [comment]
Do we have canary for this task?

**Comment by sdobrodeev** [comment]
> Do we have canary for this task?

https://github.com/microsoft/azure-pipelines-canary/pull/514

**Review by egor-bryzgalov — APPROVED** [review]

**Review by anatolybolshakov — APPROVED** [review]

File: Tasks/UseRubyVersionV0/task.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 179,
+        "Minor": 181,
```
Comment (DaniilShmelev) [inline Tasks/UseRubyVersionV0/task.json:16]: This task should be merged on [sprint 182](https://github.com/microsoft/build-task-team/milestone/7)

Comment (anatolybolshakov) [reply Tasks/UseRubyVersionV0/task.json]: Bumped task version

**Comment by anatolybolshakov** [comment]
Let's merge it after mering of related canary PR

**Review by DaniilShmelev — APPROVED** [review]

**Pull request merged**